tests/check_check_master.c

```diff
@@ -611,8 +779,8 @@ START_TEST(test_check_failure_ftypes)
     tr = tr_fail_array[i - passed];
     ck_assert_msg(tr != NULL, NULL);
     ck_assert_msg(master_tests[i].failure_type == tr_rtype(tr),
-                "Failure type wrong for test %d:%s:%s",
```

Comment:
I'm confused what is going on here. Are the tabs being replaced with spaces, or only some of the tabs?